meetings/2024-03-11/minutes.md

```diff
@@ -0,0 +1,82 @@
+# OQS Technical Steering – 2024-03-11 – minutes
+
+## Attendees
+
+TSC members
+
+- [x] [Norman Ashley (Cisco)](https://github.com/ashman-p)
+- [x] [Michael Baentsch (independent contributor)](https://github.com/baentsch)
+- [x] [Thomas Bailleux (SandboxAQ)](https://github.com/)
+- [x] [Basil Hess (IBM Research)](https://github.com/bhess)
+- [x] [Brian Jarvis (AWS)](https://github.com/brian-jarvis-aws)
+- [x] [Christian Paquin (Microsoft Research)](https://github.com/christianpaquin)
+- [x] [Douglas Stebila (University of Waterloo)](https://github.com/dstebila)
+
+Other attendees
+
+- Alex Bozarth
+- Deirdre Connolly
+- Jason Goertzen
+- Nigel Jones
+- Ry Jones
+- Michael Maximilien
+- Hart Montgomery
+- Massimiliano Pala
+- Spencer Wilson
+
+## 0. Welcome
+
+Douglas welcomes attendees to the first meeting of the OQS TSC.
+
+## 1. Election of a TSC chair
+
+Douglas opens the floor to nominations for a TSC chair.
+
+Ry recommends that the TSC appoint a chair and a co-chair, who can lead meetings in the chair's absence and assist with organization.
+
+The following names are nominated for chair or co-chair:
+
+- Douglas
+- Max
+- Basil
+- Brian
+- Thomas
+
+It is noted that the chair and co-chair need to be a voting member of the TSC, so Max is not eligible.
+
+Douglas asks each nominee if they would like to stand for chair or co-chair. The responses are:
+
+- Douglas: chair or co-chair
+- Basil: co-chair
+- Brian: co-chair
+- Thomas: co-chair
+
+Decision: For election of the chair, a roll call vote is held. Douglas is elected chair.
+
+Decision: For election of a co-chair, each TSC member is asked to send a private message to Ry Jones via Zoom chat, who will tally the votes. Ry reports that Basil received the largest vote.
```

Comment:
Hi Michael, since indeed there was not a TSC decision about the voting procedure used during that meeting, I didn't want to convey that there had been a decision about the voting procedure, and rather wanted the minutes to reflect what voting procedure was used so at least that was stated.